roles/sft-server/files/nginx.conf

```diff
@@ -0,0 +1,34 @@
+
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
```

Comment:
Can we have  a higher limit here?

Reply:
FYI moved the limit to `NGINX_WORKER_CONNECTIONS`. I asked @z-dule for some feedback on this. Based on how the `sftd` actually behaves, I 'll adjust the `1024` accordingly. Please note, that we are only talking about signaling and metrics here, anyway.

We might also need to revisit `worker_processes` depending on how `sftd` in general deals with parallelism.